### ISSUE TITLE
ReadColorMap

### ISSUE THREAD

**Issue by moritzdannhauer** [SCIInstitute/SCIRun#1211]
One of my collaborators requested a way to manipulate the Colormap.
ReadColorMap would offer an easy solution for that problem.

**Comment by dcwhite** [comment]
@jessdtate For interns?

**Comment by jessdtate** [comment]
yeah.  we need to meet to discuss the format and things.  It's very different in 5 than 4

**Comment by dcwhite** [comment]
We've been dancing around this one lately, let's just get something implemented.

**Comment by dcwhite** [comment]
See #1791
